bucket/discord.json

```diff
@@ -1,31 +1,37 @@
 {
-    "version": "0.0.311",
+    "version": "1.0.9004-11",
     "description": "Free Voice and Text Chat",
     "homepage": "https://discordapp.com/",
     "license": {
         "identifier": "Freeware",
         "url": "https://discordapp.com/terms"
     },
-    "notes": "To avoid manifest removal due to internal updates, disable automatic updates by running: '$dir\\disable-auto-update.ps1'",
-    "url": "http://dl.discordapp.net/apps/win/Discord-0.0.311-full.nupkg",
-    "hash": "sha1:3928a4af8e249184f2d4212bd8c958adc8b2ff4f",
-    "extract_dir": "lib\\net45",
-    "bin": "Discord.exe",
+    "url": "https://github.com/portapps/discord-portable/releases/download/1.0.9004-11/discord-portable-win32-1.0.9004-11.7z",
+    "hash": "670691395ba0bd2ac59e5cbaa1af72cbdae91bd44ed5ec69f922d1547b8078af",
+    "bin": [
```

Comment:
Does it accept any CLI args? If not, bin is not needed.

Reply:
Actually, it does not, so I shall remove the bin section.